modules/artifact/src/main/java/org/mule/runtime/module/artifact/api/classloader/FineGrainedControlClassLoader.java

```diff
@@ -56,11 +56,11 @@ public FineGrainedControlClassLoader(URL[] urls, ClassLoader parent, ClassLoader
     super(urls, parent, new NonCachingURLStreamHandlerFactory());
     checkArgument(lookupPolicy != null, "Lookup policy cannot be null");
     this.lookupPolicy = lookupPolicy;
-    verboseLogging = LOGGER.isDebugEnabled() || isVerboseLoggingEnabled();
+    verboseLogging = valueOf(getProperty(MULE_LOG_VERBOSE_CLASSLOADING));
   }
 
-  private boolean isVerboseLoggingEnabled() {
-    return LOGGER.isInfoEnabled() && valueOf(getProperty(MULE_LOG_VERBOSE_CLASSLOADING));
+  private boolean isVerboseLogging() {
+    return LOGGER.isDebugEnabled() || verboseLogging;
```

Comment:
this is not only repeated code, but in the original version INFO level was being used whereas now you degraded it to DEBUG

Reply:
nope, this is in the same place but is not the same method. This does the same evaluation that was previously done in the constructor, to account for log level changes. The repeated code you mention i think is with `FilteringArtifactClassLoader`. There is no common superclass between the 2 to extract this logic to, and it seems overkill to extract an utils for just 2 lines (plus i would need to pass the logger around anyway)